mifosng-android/src/main/res/values/strings.xml

```diff
@@ -733,6 +733,12 @@
     <string name="incorrect_passcode">Incorrect Passcode</string>
     <string name="incorrect_passcode_more_than_three">You have entered incorrect Passcode more than 3 times. Please login again.</string>
     <string name="error_passcode">Passcode should be of 4 digit</string>
+    <string name="gps_network_not_enabled">GPS network is disabled</string>
+    <string name="open_location_settings">Open location settings</string>
+    <string name="Cancel">Cancel</string>
```

Comment:
Also, have a look at this interesting SO thread and the mentioned blog about the Location Manager vs Play Services.

Reply:
Missed the link: https://stackoverflow.com/questions/33022662/android-locationmanager-vs-google-play-services

Reply:
I googled it! I didn't know about this alternative method. Thanks, I am reading that!
Thanks for the review! I'll make the changes asap.

Reply:
@tarun0 In that case, should [this](https://stackoverflow.com/questions/30754117/android-check-location-services-enabled-if-using-googleapiclient) be considered to check if location services are enabled? What do you suggest? Thanks!